runtime/src/chpl-file-utils.c

```diff
@@ -152,7 +152,10 @@ qioerr chpl_fs_is_link(int* ret, const char* name) {
   // comparison is also not valid when an unlinked file is provided.
   struct stat buf;
   int exitStatus = lstat(name, &buf);
-  if (exitStatus)
+  if (exitStatus == -1 && errno == ENOENT) {
+    // The link examined does not exist, return false
+    *ret = 0;
+  } else if (exitStatus)
```

Comment:
Stylistically I find this a bit hard to read, I'd prefer wrapping the return in brackets.